cheatsheets/Transport_Layer_Security_Cheat_Sheet.md

```diff
@@ -136,19 +136,17 @@ For internal applications, an internal CA can be used. This means that the FQDN
 
-If the user does not know or trust this intermediate CA then the certificate validation will fail, even if the user trusts the ultimate root CA, as they cannot establish a chain of trust between the certificate and the root. In order to avoid this, any intermediate certificates should be provided alongside the main certificate.
+Organization Validated (OV) certificates include the subscriber’s organization information in the certificates subject DN. E.g. C = GB, ST = Manchester, **O = Sectigo Limited**, CN = sectigo.com. The process to acquire an OV certificate requires official contact with the subscriber company via a method that proves to the CA that they are truly talking to the right company.
 
-### Consider the use of Extended Validation Certificates
-
-Extended validation (EV) certificates claim to provide a higher level of verification of the entity, as they perform checks that the requestor is a legitimate legal entity, rather than just verifying the ownership of the domain name like normal (or "Domain Validated") certificates. This can effectively be viewed as the difference between "This site is really run by Example Company Inc." vs "This domain is really example.org".
+Extended validation (EV) certificates claim to provide an even higher level of verification of the entity than OV checks. They perform checks that the requestor is a legitimate legal entity, rather than just verifying the ownership of the domain name like OV certificates or simply showing practical proof of control, like DV certificates. This can effectively be viewed as the difference between "This site is really run by Example Company Inc." vs "This domain is really example.org".
 
 Historically these displayed differently in the browser, often showing the company name or a green icon or background in the address bar. However, as of 2019 both [Chrome](https://groups.google.com/a/chromium.org/forum/m/#!msg/security-dev/h1bTcoTpfeI/jUTk1z7VAAAJ) and [Firefox](https://groups.google.com/forum/m/?fromgroups&hl=en#!topic/firefox-dev/6wAg_PpnlY4) have announced that they will be removing these indicators, as they do not believe that EV certificates provide any additional protection.
 
-There is no security downside to the use of EV certificates. However, as they are significantly more expensive than domain validated certificates, an assessment should be made to determine whether they provide any additional value.
+As all browsers and TLS stacks are unaware of the different between DV, OV, and EV certificates, they are effectively the same in terms of security. An attacker only need to reach the level of practical control of the domain to get a rogue certificate.  The additional pain in getting OV and EV certificates may create an availability risk and there use should be thoroughly reviewed with this in mind.
```

Comment:
"Only needs to" is slightly misleading when our web security model depends on this being sufficient.

Typo: there use.

Thoroughly reviewed could be better as "reviewed" or even "reconsidered" depending on the intended tone. (Simpler language)

This could be good paragraph for example about company name confusion like the Stripe, Inc. example I shared.

My understanding is that outside browsers, the subject fields other than CN are only used for code signing? In this context I think only browsers and APIs are relevant though.

Reply:
> My understanding is that outside browsers, the subject fields other than CN are only used for code signing? In this context I think only browsers and APIs are relevant though.

Maybe for S/MIME too, though I more often used PGP-mail because for the few I've exchanged encryption or signed email, exchanging GPG keys was just easier than getting certs for S/MIME. But I digress.

Getting back to the context of TLS server-side X.509v3 certificates, I think that the SubjectAltName (aka, SAN) extension, if set, is supposed to take precendence over the CN. There's probably other fields that are non-critical extensions that are important as well that I'm forgetting, but it's been so long since I read through the PKIX RFC that I'll defer to @markgamache, as SAN is the only one I remember off the top of my head. Also, I think its a bit different client-side certs used with mTLS, but I'm not sure we want to open that can of worms.

Reply:
Right. When I said "subject", I meant who the certificate is issued to.

I don't think CN or SAN needs to be explicitly mentioned either way. Where I said CN, read it as SAN.

Reply:
> "Only needs to" is slightly misleading when our web security model depends on this being sufficient.
> 
> Typo: there use.
> 
> Thoroughly reviewed could be better as "reviewed" or even "reconsidered" depending on the intended tone. (Simpler language)
> 
> This could be good paragraph for example about company name confusion like the Stripe, Inc. example I shared.
> 
> My understanding is that outside browsers, the subject fields other than CN are only used for code signing? In this context I think only browsers and APIs are relevant though.

I changed only to simply. Not sure that is a lot better. I also don't have a clear understanding of how non-Americans would take only or simply. The intent is not to make DV seem easy, simply easier than OV and EV. I am open to a better way to say it. Leaving out only or simply seems to not make the point as well, in my mind. 

Taking the other edits as well.